crates/red_knot_python_semantic/src/types/display.rs

```diff
@@ -180,24 +180,24 @@ impl Display for DisplayLiteralGroup<'_> {
 }
 
 #[derive(Copy, Clone, Debug, Eq, PartialEq, Hash)]
```

Comment:
```suggestion
/// Enumeration of literal types that are displayed in a "condensed way" inside `Literal` slices.
///
/// For example, `Literal[1] | Literal[2]` is displayed as `"Literal[1, 2]"`.
/// Not all `Literal` types are displayed using `Literal` slices
/// (e.g. it would be inappropriate to display `LiteralString`
/// as `Literal[LiteralString]`).
#[derive(Copy, Clone, Debug, Eq, PartialEq, Hash)]
```